src/WebJobs.Script/Description/DotNet/RuntimeAssembliesInfo.cs

```diff
@@ -34,10 +35,10 @@ public RuntimeAssembliesInfo(IEnvironment instance)
             lock (_loadSyncRoot)
             {
                 _relaxedUnification = FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagRelaxedAssemblyUnification, _environment);
-
-                string manifestName = _relaxedUnification.Value
+                var tfmversion = typeof(string).Assembly.GetName().Version.Major;
```

Comment:
Will revisit this flow, but this will give us the resolution we need at the moment.